packages/target-tools/delivery-api-client/models/DeliveryRequest.d.ts

```diff
@@ -3,7 +3,7 @@ export interface DeliveryRequest {
     requestId?: string;
     impressionId?: string;
     id?: VisitorId;
-    environmentId?: number;

```

Comment:
Did you mean to remove `environmentId` ?  I don't think I should have been replaced by `organizationId`.  Both should be permissible.

Reply:
Yes, it was intentional, to be consistent with openapi-specs since `organizationId` is sufficient to identify a client without needing `environmentId`.